src/web/mjs/connectors/VizShonenJump.mjs

```diff
@@ -0,0 +1,200 @@
+import Connector from '../engine/Connector.mjs';
+const EXIF = require('exif-js');
+
+export default class VizShonenJump extends Connector {
+
```

Comment:
There is no implementation of `async _getMangaFromURI(uri)`, therefore copy & paste will not work.